pycstruct/pycstruct.py

```diff
@@ -316,8 +512,13 @@ def add(self, datatype, name, length=1, byteorder="", same_level=False):
         :param name: Name of element. Needs to be unique.
         :type name: str
         :param length: Number of elements. If > 1 this is an array/list of
-                       elements with equal size. Default is 1.
+-                      elements with equal size. Default is 1. This should only
```

Comment:
This line starts with -, thats why the doc generation gives warnings and the doc has strange indententation.

Reply:
Oups... Thanks for the check.